tests/utils.spec.js

```diff
@@ -209,4 +211,47 @@ describe('Utils', () => {
       }));
     });
   });
+
+  describe('startsWith', () => {
+
+    it('returns true if search string is at start', () => {
+      expect(startsWith('example', 'ex')).toBe(true);
+    });
+
+    it('returns false if search string is not at start', () => {
+      expect(startsWith('example', 'no')).toBe(false);
+    });
+
+    it('returns false if search string is not at start, even if present', () => {
+      expect(startsWith('example', 'amp')).toBe(false);
+    });
+  });
+
+  describe('endsWith', () => {
+
+    it('returns true is search string is at end', () => {
+      expect(endsWith('example', 'le')).toBe(true);
+    });
+
+    it('handles a string with multiple occurences', () => {
+      expect(endsWith('siryessir', '')).toBe(true);
```

Comment:
Shouldn't the test case be `endsWith('siryessir', 'sir')?`

Reply:
Whoops! Yes, that's what I intended. Pushing an update now.